package.json

```diff
@@ -37,7 +37,7 @@
     "@ministryofjustice/frontend": "^2.1.3",
```

Comment:
Not specific to this line, but remember to increase version number of the app in package.json